### DISCUSSION TITLE
`INPUT_LINE_NUMBER` but plus half terminal height?

### DISCUSSION THREAD

**Discussion by J-Kappes** [kovidgoyal/kitty#6873] [Q&A] [answered]
I was considering adding the argument `-j.5` to the `less` scrollback pager in order to have search results appear at the center of the screen instead of the top. However, this also affects the `+` argument, messing up the alignment with your terminal state before `less`. Any ideas on how to make this work?

**Comment by kovidgoyal** [comment]
Use a wrapper script for less as your pager that does whatever
manipulations you like on the arguments to less

**Comment by J-Kappes** [comment, marked as answer]
That works. I made my `scrollback_pager` `kittyless INPUT_LINE_NUMBER` with kittyless being
```sh
#!/bin/sh
offset=`expr $1 + $(tput lines) / 2 - 1`
less -j.5 --chop-long-lines --RAW-CONTROL-CHARS +$offset
```
